eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/ManualReferenceTestRunner.java

```diff
@@ -38,16 +39,30 @@ public class ManualReferenceTestRunner extends Eth2ReferenceTestCase {
    *
    * <p>e.g. set to "ssz_static" to run only ssz static tests or "ssz_static/Attestation" for only
    * attestation ssz tests.
+   *
+   * <p>May be overridden by the ENV_TEST_TYPE environment variable.
    */
   private static final String TEST_TYPE = "fork_choice";
 
-  /** Filter test to run to those from the specified spec. One of general, minimal or mainnet */
+  /**
+   * Filter test to run to those from the specified spec. One of general, minimal or mainnet
+   *
+   * <p>May be overridden by the ENV_SPEC environment variable.
+   */
   private static final String SPEC = "";
 
-  /** Filter test to run only those for a specific milestone. Use values from TestFork. */
-  private static final String MILESTONE = null;
+  /**
+   * Filter test to run only those for a specific milestone. Use values from TestFork.
+   *
+   * <p>May be overridden by the ENV_MILESTONE environment variable.
+   */
```

Comment:
i like this, but the one other thing i'd add is the default to run all... so milestone would be null to run all, but some are empty string etc...

Reply:
`MILESTONE = ""` will run all milestones by default 👍